### ISSUE TITLE
Icon für Hydranten-Layer in Layerauswahl ändern

### ISSUE THREAD

**Issue by sebkur** [waldbrand/website#21]
Das Icon ist zur Zeit der "unbekannte Hydrant", das ist nicht optimal

**Comment by mozita** [comment]
Ist das Fragezeichen Icon überhaupt noch sinnvoll, bzw. wann wird es angezeigt? Aktuell sehe ich nur das normale Hydranten Icon, auch wenn die Angaben im editor nicht gemacht sind. Wenn keine Angaben gemacht sind, dann signalisiert dies das Fragezeichen Icon ja ganz gut. 

Ist es zu kompliziert den Editor so einzustellen, dass er das Icon je nach dem auswählt, ob weitere Eingaben zum Hydrant gemacht werden oder nicht?

**Comment by sebkur** [comment]
ja, doch, das Icon erscheint, wenn ein Hydrant eingetragen ist, aber der Type nicht angegeben wurde.
Bsp. hier: https://waldbrand-app.de/mapcomplete/waldbrand.html?z=16&lat=51.79754&lon=14.33029#

**Comment by sebkur** [comment]
> Ist es zu kompliziert den Editor so einzustellen, dass er das Icon je nach dem auswählt, ob weitere Eingaben zum Hydrant gemacht werden oder nicht?

ich verstehe jetzt noch nicht genau, was du meinst. Gerade wird es angezeigt, wenn der typ nicht angegeben ist.

Im Prinzip wäre es auch cool, wenn wir signalisieren würden, wo noch was ergänzt werden kann, also zum Beispiel Innendurchmesser bei Hydranten oder das Volumen bei Wasserspeichern fehlt

**Comment by mozita** [comment]
achso ja, gut. Das Fragezeichen kommt nur bei unbekanntem Typ. Ja, dann ist es ja so erstmal voll ok.
